build.xml

```diff
@@ -450,6 +450,10 @@
           <dependency groupId="io.netty" artifactId="netty-all" version="4.0.44.Final" />
           <dependency groupId="com.google.code.findbugs" artifactId="jsr305" version="2.0.2" />
           <dependency groupId="com.clearspring.analytics" artifactId="stream" version="2.5.2" />
+          <dependency groupId="com.palantir.tracing" artifactId="tracing" version="4.23.0" />
+          <dependency groupId="com.palantir.tracing" artifactId="tracing-api" version="4.23.0" />
+          <dependency groupId="com.palantir.safe-logging" artifactId="preconditions" version="1.8.2" />
+          <dependency groupId="com.palantir.safe-logging" artifactId="safe-logging" version="1.8.2" />
```

Comment:
I'm surprised we don't need the safe-logging:logger dependencies, iirc the `tracing` library uses it